RealmSwift/Sync.swift

```diff
@@ -176,6 +176,37 @@ public typealias SyncLogLevel = RLMSyncLogLevel
  */
 public typealias Provider = RLMIdentityProvider
 
+/**
+ * How the SSL certificate for the Realm Object Server should be validated.
+ *
+ * By default, when connecting to the Realm Object Server over HTTPS, Realm will
+ * validate the server's HTTPS certificate using the system trust store and root
+ * certificates. For additional protection against man-in-the-middle (MITM)
+ * attacks and similar vulnerabilities, you can pin a certificate or public key,
+ * and reject all others, even if they are signed by a trusted CA.
+ */
+public enum ServerValidationPolicy {
```

Comment:
This controls how the client validates that the server is who it claims to be. `.none` doesn't disable SSL; it just disables validation of the certificate (which is a pre-existing option).

Reply:
Good point, I was thinking about this backwards.

Reply:
Probably worth rewording the docs a bit to make sure that's clear, since I guess it doesn't outright say it's controlling how the client validates the server.